easybuild/easyconfigs/n/ncurses/ncurses-6.3-GCCcore-12.1.0.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'ConfigureMake'
+
+name = 'ncurses'
+version = '6.3'
+
+homepage = 'https://www.gnu.org/software/ncurses/'
+description = """
+ The Ncurses (new curses) library is a free software emulation of curses in
+ System V Release 4.0, and more. It uses Terminfo format, supports pads and
+ color and multiple highlights and forms characters and function-key mapping,
+ and has all the other SYSV-curses enhancements over BSD Curses.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.1.0'}
```

Comment:
This same `ncurses` easyconfig is also included in #16076, but some changes are needed because of recent changes in the `develop` branch, see https://github.com/easybuilders/easybuild-easyconfigs/pull/15903 

The easiest way forward is probably to update the `ncurses` easyconfig in #16076 first, get that PR merged, and then sync up this PR with `develop` once that's done.

Reply:
@boegel can you delete this file since the new EC is already merged in `develop` branch?